app/components/application/browse/application-list.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, Input, OnDestroy, OnInit, ViewChild } from "@angular/core";
+import { Router } from "@angular/router";
+import { autobind } from "core-decorators";
+import { List } from "immutable";
+import { Observable, Subscription } from "rxjs";
+
+import { BackgroundTaskManager } from "app/components/base/background-task";
+import { LoadingStatus } from "app/components/base/loading";
+import { QuickListComponent, QuickListItemStatus } from "app/components/base/quick-list";
+import { ListOrTableBase } from "app/components/base/selectable-list";
+import { TableComponent } from "app/components/base/table";
+import { Application } from "app/models";
+import { AccountService, ApplicationService } from "app/services";
+import { RxListProxy } from "app/services/core";
+import { Filter } from "app/utils/filter-builder";
+import { DeleteApplicationAction } from "../action";
+
+@Component({
+    selector: "bex-application-list",
+    templateUrl: "./application-list.html",
+})
+export class ApplicationListComponent extends ListOrTableBase implements OnInit, OnDestroy {
+    public status: Observable<LoadingStatus>;
+    public data: RxListProxy<{}, Application>;
+    public applications: List<Application>;
+    public displayedApplications: List<Application>;
+
+    @ViewChild(QuickListComponent)
```

Comment:
what, both of these? or just the @ViewChild bit? 

    @ViewChild(QuickListComponent)
    public list: QuickListComponent;

    @ViewChild(TableComponent)
    public table: TableComponent;